Assets/Engine/Interactions/IntentManager.cs

```diff
@@ -19,13 +19,27 @@ public class IntentManager : MonoBehaviour
     public Color colorHarm;
     public Color colorHelp;
 
+    private Button IntentButton;
+
     public enum IntentType
     {
         Help,
         Harm
     }
+
+    void Start()
+    {
+        IntentButton = GetComponent<Button>();
+        IntentButton.onClick.AddListener(TaskOnClick);
+    }
+
+    void TaskOnClick()
```

Comment:
Braces should be aligned similarly to the other methods. These ones are indented a bit too far.